### ISSUE TITLE
Cache corruption issue with useQuery

### ISSUE THREAD

**Issue by iankberry** [appmotion/apollo-augmented-hooks#7]
I've got another strange issue where the cached data coming back from `useQuery` is slightly corrupted. It's the same query that I was referencing in #6.

Here is what the data looks like when using the stock `@apollo/client` version of `useQuery`:
```json
{
  "selectedEnergyEvent": {
    "__typename": "EnergyEvent",
    "id": "8357862188871433499",
    "time": "2021-07-28T18:39:08-04:00",
    "startTime": "2021-07-28T14:34:31-04:00",
    "endTime": "2021-07-28T18:39:08-04:00",
    "energyType": "DRAIN",
    "algorithm": "WORK_TIME",
    "algorithmData": null,
    "context": [
      {
        "__typename": "EnergyActivityContext",
        "contextType": "ACTIVITY",
        "activity": {
          "__typename": "ActivitySlice",
          "id": "6100042703173699744",
          "providerId": "504326868638495413",
          "providerType": "GOOGLE",
          "providerGroupName": "Default",
          "startTime": "2021-07-28T14:34:31-04:00",
          "endTime": "2021-07-28T14:38:32-04:00",
          "activityType": "APP",
          "activityGroup": "iTerm",
          "activityCount": 2,
          "audience": "INDIVIDUAL",
          "categories": [
            "WORKGROUP_ACCOUNT"
          ],
          "participants": [],
          "energyType": "NONE",
          "vendorUrl": null
        }
      }
    ]
  }
}
```

And the exact same query with `apollo-augmented-hooks`:
```json
{
  "selectedEnergyEvent": {
    "id": "8357862188871433499",
    "__typename": "EnergyEvent",
    "time": "2021-07-28T18:39:08-04:00",
    "startTime": "2021-07-28T14:34:31-04:00",
    "endTime": "2021-07-28T18:39:08-04:00",
    "energyType": "DRAIN",
    "algorithm": "WORK_TIME",
    "algorithmData": null,
    "context": [
      {
        "__typename": "EnergyActivityContext",
        "contextType": "ACTIVITY",
        "activity": {
          "id": "6100042703173699744",
          "__typename": "ActivitySlice",
          "providerId": "504326868638495413",
          "providerType": "GOOGLE",
          "providerGroupName": "Default",
          "startTime": "2021-07-28T14:34:31-04:00",
          "endTime": "2021-07-28T14:38:32-04:00",
          "activityType": "APP",
          "activityGroup": "iTerm",
          "activityCount": 2,
          "audience": "INDIVIDUAL",
          "categories": [
            {
              "0": "W",
              "1": "O",
              "2": "R",
              "3": "K",
              "4": "G",
              "5": "R",
              "6": "O",
              "7": "U",
              "8": "P",
              "9": "_",
              "10": "A",
              "11": "C",
              "12": "C",
              "13": "O",
              "14": "U",
              "15": "N",
              "16": "T"
            }
          ],
          "participants": [],
          "energyType": "NONE",
          "vendorUrl": null
        }
      }
    ]
  }
}
```

Possible relevant, I am using the `apollo-cache-persist` library to persist my Apollo cache to localstorage. If I look at the data in localstorage directly, it is correctly formatted though.

If I had to guess, it looks like a possible issue with the spread operator on a nested array but you'd know better than me :-)

**Comment by mindnektar** [comment]
Hey, I just fixed this in v1.11.4 - I happened to stumble across the very thing in one of my projects today. Thanks again for helping me make this library as good as possible!

**Comment by iankberry** [comment]
Worked like a charm. Thanks again!
